02-express-tutorial/package.json

```diff
@@ -10,7 +10,7 @@
   "author": "",
   "license": "ISC",
   "dependencies": {
-    "express": "^4.17.1"
+    "express": "^4.18.2"
```

Comment:
What are the chances that a change like this will cause an issue? How do you know?